examples/policygen/generated/forseti_policies/policies/templates/gcp_iam_restrict_service_account_key_age_v1.yaml

```diff
@@ -67,16 +67,23 @@ spec:
           	metadata := {"resource": asset.name}
           }
           
-          check_key_not_expired(key) = check_key_not_expired {
+          check_key_not_expired(key) {
           	expiry_time := time.parse_rfc3339_ns(lib.get_default(key, "validBeforeTime", "1900-01-01T01:00:006Z"))
+          	expiry_time >= 0
           	now := time.now_ns()
-          	check_key_not_expired := now < expiry_time
+          	now < expiry_time
           }
           
-          check_key_age(key, max_age) = check_key_age {
+          # Workaround for dates in the far future - https://github.com/open-policy-agent/opa/issues/4098
```

Comment:
Were the changes in this file generated from your changes in the HCL files?

Reply:
No, my changes did not touch policies. `scripts/regen.sh` updated this file.